.github/workflows/pull-request-checks.yaml

```diff
@@ -118,7 +118,7 @@ jobs:
         run: ccache -z --max-size=500M
       - name: Build with make
         run: |
-          make -C src       -j2
+          make -C src       -j2 MINISAT2=../../minisat-2.2.1 CADICAL=../../cadical
```

Comment:
I am probably wrong about this but shouldn't the assignment of environmental variables be before the make, as in:

```
CXXFLAGS="-O0 -g" make -j4000
```

Reply:
AFAIK, this is a standard way for passing variables from the command-line to make (e.g. see http://books.gigatux.nl/mirror/cinanutshell/0596006977/cinanut-CHP-19-SECT-5.html). In this form, the variables are not treated as environment variables, and they are parsed by the `make` command.